src/main/java/uk/ac/ebi/eva/pipeline/model/converters/data/VariantToMongoDbObjectConverter.java

```diff
@@ -44,21 +45,20 @@
 
     private boolean includeStats;
 
-    public VariantToMongoDbObjectConverter(boolean includeStats, Map<String, Integer> samplesPosition,
-                                           boolean calculateStats, boolean includeSample,
+    public VariantToMongoDbObjectConverter(boolean includeStats, boolean calculateStats, boolean includeSample,
                                            VariantStorageManager.IncludeSrc includeSrc) {
+        
         this.includeStats = includeStats;
         this.statsConverter = calculateStats ? new DBObjectToVariantStatsConverter() : null;
-        DBObjectToSamplesConverter sampleConverter =
-                includeSample ? new DBObjectToSamplesConverter(true, samplesPosition) : null;
+        DBObjectToSamplesConverter sampleConverter = includeSample ? new DBObjectToSamplesConverter() : null;
         this.sourceEntryConverter = new DBObjectToVariantSourceEntryConverter(includeSrc, sampleConverter);
         this.variantConverter = new DBObjectToVariantConverter(null, null);
     }
 
     @Override
     public DBObject convert(Variant variant) {
         Assert.notNull(variant, "Variant should not be null. Please provide a valid Variant object");
-        logger.debug("Convert variant {} into mongo object", variant);
+        logger.trace("Convert variant {} into mongo object", variant);
```

Comment:
that logs every variant that is converted, I can see that we will use the debug level sometimes, but this would be overkill.